.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-versions: [1.12.x, 1.13.x, 1.14.x]
+        platform: [ubuntu-latest, macos-latest]
```

Comment:
do we care about `windows` too here?
i'd probably use `ubuntu-18.04, macos, windows`
the macos and windows don't seem to have versions as part of the names.

Reply:
Let's do windows in a subsequent patch @neolit123 i kicked the tires and did not get a clean run. Here's a clean run with the changes in this PR:
https://github.com/dims/klog/actions/runs/73799189

( windows failures that we can chase later - https://github.com/dims/klog/actions/runs/73797937 )

Reply:
let's pin if we see a problem with -latest please